test/k8sT/Services.go

```diff
@@ -1552,7 +1552,7 @@ Secondary Interface %s :: IPv4: (%s, %s), IPv6: (%s, %s)`, helpers.DualStackSupp
 	SkipContextIf(func() bool {
 		// The graceful termination feature depends on enabling an alpha feature
 		// EndpointSliceTerminatingCondition in Kubernetes.
-		return helpers.SkipK8sVersions("<1.20.0") || helpers.RunsOnGKE() || helpers.RunsOnEKS() || helpers.SkipQuarantined()
+		return helpers.SkipK8sVersions("<1.20.0") || helpers.RunsOnGKE() || helpers.RunsOnEKS()
```

Comment:
Can we wait for 10 runs in CI before removing from quarantine as discussed in https://github.com/cilium/cilium/pull/18051#issuecomment-982429496 and https://github.com/cilium/cilium/pull/18051#issuecomment-982904054?

Reply:
We already have 9 runs on this PR. I'll trigger one more after I address your comment below.